sql/dd/impl/system_registry.cc

```diff
@@ -246,6 +246,12 @@ void System_tables::add_remaining_dd_tables() {
   register_table("time_zone_transition_type", system);
   register_table("user", system);
 
+#ifdef WITH_WSREP
+  register_table("wsrep_cluster", system);
+  register_table("wsrep_cluster_members", system);
+  register_table("wsrep_streaming_log", system);
```

Comment:
These are non-core DD tables and called as 'system tables'. So DD version bump is not necessary.
A side note:
As a downstream, we cannot bump the DD version at all because it will cause upgrade incompatibilities (upgrade from MySQL server).

We could have added another version number but a bit trickier to maintain and keep in sync.

Reply:
Thank you for checking.